colossus-docs/src/test/scala/TestkitExampleSpec.scala

```diff
@@ -0,0 +1,18 @@
+import colossus.protocols.http.HttpRequest
+import colossus.testkit.CallbackAwait
+import colossus.testkit.MockConnection
+import org.scalatest.{MustMatchers, WordSpec}
+
+import scala.concurrent.duration._
+
+// #example
+/*class TestkitExampleSpec extends WordSpec with MustMatchers {
```

Comment:
I couldn't get this example on teskit (http://tumblr.github.io/colossus/docs/testkit/) to work. Help!